coderd/files/cache.go

```diff
@@ -43,15 +98,30 @@ type Cache struct {
 	lock sync.Mutex
 	data map[uuid.UUID]*cacheEntry
 	fetcher
+
+	// metrics
+	currentOpenFileReferences prometheus.Gauge
+	totalOpenFileReferences   prometheus.Counter
+
+	currentOpenFiles prometheus.Gauge
+	totalOpenedFiles prometheus.Counter
+
+	currentCacheSize prometheus.Gauge
+	totalCacheSize   prometheus.Counter
+}
+
+type cacheEntryValue struct {
+	dir  fs.FS
```

Comment:
```suggestion
	fs.FS
```

`dir` sounds like a path or something, I've just let it be inferred as `FS` in a few places